plugins/statistics.koplugin/main.lua

```diff
@@ -1074,6 +1074,52 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                         end,
                         separator = true,
                     },
+                    {
+                        text_func = function()
+                            -- @translators %1 is the time in the format 00:00
+                            return T(_("Calendar days start at %1 "),
```

Comment:
Not `Timeline calendar day(s) start(s) at 04:00` ? to make it more obvious it applies mainly there, and then elsewhere if `[x] Also use in calendar view` ? Or too long, an short form is just better looking vs. the other items?

Reply:
Changed to "Daily timeline starts at".